src/hugr/views/hierarchy.rs

```diff
@@ -50,6 +50,15 @@ where
     hugr: &'g Base,
 }
 
+impl<'g, Base> SiblingGraph<'g, Base>
+where
+    Base: HugrInternals,
+{
+    pub(super) fn base(&self) -> &Base {
```

Comment:
Does this actually give you anything over just `pub(super)`-ing the `hugr` field? I haven't grokked what's going on with lifetimes yet but I think this just narrows the lifetime of the `self.hugr` reference, right?

Reply:
Done!